classes.py

```diff
@@ -20,6 +21,31 @@ def __init__(self):
     self.enemy_size = 50
 
     self.alive_enemies = []
+    self.state_of_game = False # False = waiting for user to start game, True = shooting down enemies
```

Comment:
Good thing you added the comment, because otherwise this was way too confusing to understand.
Except that in this case the solution would be to name it properly so that a comment isn't needed. This isn't exactly a complicated concept that can only be solved with a comment.